index.html

```diff
@@ -70,10 +83,13 @@ <h3 class="s3h3">Multi-Post Stories</h3>
             </li>
           </ul>
 
-          <button class="btn1">See Project</button>
+         <button class="btn1">See Project</button>
```

Comment:
#### As asked by the previous Code Reviewer, Any of your See Project buttons are not working. You should follow the button styling: Active, Hover, and Pressed. Please try to match this button according to the Figma design asked by the previous reviewer.

Reply:
can U tell me how can I fix it because I tried to add a tag and linters read it as an error